lexicons/app/bsky/graph/starterpack.json

```diff
@@ -0,0 +1,50 @@
+{
+  "lexicon": 1,
+  "id": "app.bsky.graph.starterpack",
+  "defs": {
+    "main": {
+      "type": "record",
+      "description": "Record defining a starter pack of actors and feeds for new users.",
+      "key": "any",
+      "record": {
+        "type": "object",
+        "required": ["name", "createdAt"],
+        "properties": {
+          "name": {
+            "type": "string",
+            "maxLength": 64,
```

Comment:
Cool if we drop this down to 50?

Reply:
The main consideration is that it would be an exception from other display names, e.g.
 - lists
 - profiles
 - labels

If that seems fine to yall then sounds good!